src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/ComponentSpec/config.jelly

```diff
@@ -8,9 +8,11 @@
         <f:entry title="Initial Job" field="firstJob">
             <f:select/>
         </f:entry>
+        <f:entry title="Final Job" field="lastJob">
```

Comment:
I would like it to be clear that this is an optional attribute. What do you think of adding a "(optional)" afterwards?